src/Components/ViewProject/ViewProject.tsx

```diff
@@ -17,29 +19,30 @@ useEffect(()=>{
   axios.get(`${baseUrl}/project/${id}`,{headers:reqHeaders}).then((response)=>{
     setProject(response?.data)
   }).catch((error)=>{
-    console.log(error);
+   
+    getToastValue('error',error.response.data.message)
   })
 },[])
   return (<>
 
-<div className="container">  
+<div className="container ">  
     <div className="col-md-12">
       <h3>Project Board</h3>
     </div>
     <div className="col-md-8 m-auto ">
       <div className=" col-md-8  m-auto content  bg-view rounded-3 text-center vh-50 mt-5">
             <div className='pt-5 mb-5 mt-5'>
             <img src={avatar} alt="" className='pb-3'/>
-        <h5 className='text-black'>  {Project?.manager?.userName}  </h5>
+        <h5 className='text-white'>  {Project?.manager?.userName}  </h5>
         </div>
         <div className="secondary mb-3 pb-3 view ">
       <div className='d-flex ms-5'>
-      <label htmlFor="title   ">Title: </label>
-       <input type="text"name='title' className=' text-center form-control w-75 mb-3 m-auto rounded-5 border-0 text-white'value={Project?.title} />
+      <label htmlFor="title " className='text-white'>Title: </label>
+       <input type="text"name='title' className=' text-center form-control w-75 mb-3 m-auto rounded-5 border-0 text-black'value={Project?.title} />
```

Comment:
The input is used to take data from a user, but here we are just using it to present data. When the user first see the UI, the first thing that will be thought of is that is an input field.

You can make it disabled that will not show the writing indicator when hovering over the input and you can also change the styles to a grey color similar to that:
![image](https://github.com/FatmaHossam5/project-managment-system-butterCup-/assets/64327685/473f55f1-07b8-40bd-bcd8-91713c15e5e5)

You can also change it to div.